boards/xtensa/esp32s2_lolin_mini/doc/index.rst

```diff
@@ -0,0 +1,92 @@
+.. _esp32s2_lolin_mini:
+
+ESP32-S2
+########
+
+Overview
+********
+
+ESP32-S2 is a highly integrated, low-power, single-core Wi-Fi Microcontroller SoC, designed to be secure and
+cost-effective, with a high performance and a rich set of IO capabilities. [1]_
+
```

Comment:
While I don't think it's 100% mandatory, we tend to include a photo for each board.

Reply:
I was going to ask the same :)

Reply:
Done.